variables.tf

```diff
@@ -1,37 +1,40 @@
 variable "additional_endpoint_arns" {
   description = "Any alert endpoints, such as autoscaling, or app scaling endpoint arns that will respond to an alert"
   default     = []
-  type        = "list"
+  type        = list(string)
 }
 
 variable "sns_topic_arn" {
   description = "An SNS topic ARN that has already been created. Its policy must already allow access from CloudWatch Alarms, or set `add_sns_policy` to `true`"
   default     = ""
-  type        = "string"
+  type        = string
 }
 
 variable "add_sns_policy" {
   description = "Attach a policy that allows the notifications through to the SNS topic endpoint"
   default     = "false"
-  type        = "string"
```

Comment:
Sorry for the small change request, but can change this variable to `sns_policy_enabled` and make it a `bool` type?